example/dj/apps/app/tests/utils.py

```diff
@@ -49,7 +49,7 @@ def test_otp_generator():
     return '12345'
 
 
-def auth_token_age_test(request):
+def auth_token_age_test(request, user, auth_token):
```

Comment:
so here you get either User or subclass of User, imho my original solution was better, let it fall through to the default and let the middleware update the expiration

Reply:
nevermind, it is an edge case